### PR TITLE
8272553: several hotspot runtime/CommandLine tests don't check exit code

### PR DESCRIPTION
**Pull request by iignatev**
Hi all,

could you please review this patch that adds exit code check to several hotspot `runtime/CommandLine` tests?

Thanks,
-- Igor

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8272553](https://bugs.openjdk.java.net/browse/JDK-8272553): several hotspot runtime/CommandLine tests don't check exit code

### Reviewers
 * [David Holmes](https://openjdk.java.net/census#dholmes) (@dholmes-ora - **Reviewer**) ⚠️ Review applies to 2c8bd63cfadb815d035899d90e25e64286b3fea4

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5191/head:pull/5191` \
`$ git checkout pull/5191`

Update a local copy of the PR: \
`$ git checkout pull/5191` \
`$ git pull https://git.openjdk.java.net/jdk pull/5191/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5191`

View PR using the GUI difftool: \
`$ git pr show -t 5191`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5191.diff">https://git.openjdk.java.net/jdk/pull/5191.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/runtime/CommandLine/ConfigFileWarning.java [hunk 1]
```diff
@@ -48,6 +48,7 @@ public static void main(String[] args) throws Exception {
 
         pb = ProcessTools.createJavaProcessBuilder("-XX:Flags=hs_flags.txt","-version");
         output = new OutputAnalyzer(pb.start());
+        output.shouldNotHaveExitValue(0);
         output.shouldContain("Unrecognized VM option 'aaa'");
         output.shouldHaveExitValue(1);
```
Comment (dholmes-ora) [inline test/hotspot/jtreg/runtime/CommandLine/ConfigFileWarning.java:53]: We are already testing the exit code.

**Review by dholmes-ora — APPROVED** [review]
Hi Igor,

One test already has the check, but otherwise okay.

Thanks,
David

File: test/hotspot/jtreg/runtime/CommandLine/ConfigFileWarning.java (hunk 1 above)
Comment (iignatev) [reply test/hotspot/jtreg/runtime/CommandLine/ConfigFileWarning.java]: thanks for spotting that, removed.

**Comment by iignatev** [comment]
Thank you, David.

I've removed the redundant check from ConfigFileWarning.

-- Igor


/integrate

**Pull request closed without merging**